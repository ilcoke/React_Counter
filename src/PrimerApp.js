import React from 'react';

const PrimerApp = ({saludo = 'hola mundo'}) => {
    return (
        <>
            <h1>{saludo}</h1>
        </>
    );
}

export default PrimerApp;