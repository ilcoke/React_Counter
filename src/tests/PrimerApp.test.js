import React from 'react';
import "@testing-library/jest-dom"
import {shallow} from 'enzyme'
import PrimerApp from '../PrimerApp'


describe('Revisa componente primer app', () => {
    test('should show <primerApp />', () => {
        const saludo = 'Hola Terricola';
        const wrapper = shallow(<PrimerApp saludo ={saludo}/>);
        expect( wrapper ).toMatchSnapshot()
    })
    
})
