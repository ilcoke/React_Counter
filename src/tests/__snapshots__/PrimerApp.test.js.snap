// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Revisa componente primer app should show <primerApp /> 1`] = `
<Fragment>
  <h1>
    Hola Terricola
  </h1>
  <h2>
    subtitulo zi zi
  </h2>
</Fragment>
`;
